source/includes/stackpath/_delivery_rules.md

```diff
@@ -101,33 +99,53 @@ Required| &nbsp;
 `conditions`<br/>*Array[[Condition](#stackpath-condition)]* | At least one condition.
 `actions`<br/>*Array[[Action](#stackpath-action)]* | At least one action.
 
-#### Condition
+<!-------------------- UPDATE A DELIVERY RULE -------------------->
 
-Required| &nbsp;
-------------------------| -----------
-`trigger`<br/>*Enum* | Trigger for the condition. Possible values: `COOKIE`, `HEADER`, `HTTP_METHOD`, `STATUS_CODE`, and `URL`.
-`operator`<br/>*Enum* | Operator for the condition. Possible values: `EQUALS`, `MATCHES`, and `MATCHES_REGULAR_EXPRESSION`.
-`target`<br/>*String* | The target of the condition.
-`httpMethods`<br/>*Array[Enum]* | HTTP methods for the condition. Possible values are: `GET`, `POST`, `PUT`, `DELETE`, `HEAD`, `PATCH`, and `OPTIONS`. This fields is only used when `trigger` is set to `HTTP_METHOD`.
+#### Update a delivery rule
 
-#### Action
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/deliveryrules/d065db45-1eba-4c62-a017-d51f2d473d4a?siteId=f9dea588-d7ab-4f42-b6e6-4b85f273f3db"
+```
 
-The action attributes needed to be passed will vary depending on the condition being specified.
+> Request body example for updating a delivery rule:
 
-Possible Attributes| &nbsp;
+```js
+{
+  "conditions": [
+  {
+    "trigger": "HEADER",
+    "operator": "MATCHES",
+    "target": "my-header"
+  }],
+  "actions": [
+    {
+      "actionType": "REDIRECT",
+      "redirectUrl": "https://my-url.com"
+    }]
+}
+```
+
+> The above commands return a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/deliveryrules/:ruleId?siteId=<a href="#stackpath-sites">:siteId</a></code>
```

Comment:
This should be PUT.

Reply:
Wow! 🤦🏻 Changing this ASAP. Stand by.

Reply:
@jasondias9 Fixed in 60bd74b6b9096f130ec4e5cd60970df5ec6f7e9b